winui-toc.html

```diff
@@ -382,6 +382,7 @@
             <li><a href="/winui/Masked-TextBox/customization">Customization</a></li>
             <li><a href="/winui/Masked-TextBox/events">Events</a></li>
             <li><a href="/winui/Masked-TextBox/how-to">How to</a></li>
+			<li><a href="/winui/Masked-TextBox/error-types">Error Types</a></li>
```

Comment:
@GilbertmanojSF3971 Add new page above the customization with the heading error indication

Reply:
@Manivannan-E Sir, Modified the UG documentation as per your concern.